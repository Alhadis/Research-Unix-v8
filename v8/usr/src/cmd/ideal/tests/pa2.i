.IS
...libfile arc
triangle {
var a, b, c;
a = (-1,0);
b = (0,1);
c = (1,0);
bdlist = a, b, c;
}
.IE

.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .25;
center = (0,.5);
}
put triangle {
opaque interior;
}
'version 0' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .25;
center = (0,.5);
}
put triangle {
opaque exterior;
}
'version 1' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .5;
center = (0,.5);
}
put triangle {
opaque interior;
}
'version 2' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .5;
center = (0,.5);
}
put triangle {
opaque exterior;
}
'version 3' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .75;
center = (0,.5);
}
put triangle {
opaque interior;
}
'version 4' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .75;
center = (0,.5);
}
put triangle {
opaque exterior;
}
'version 5' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = 1;
center = (0,.5);
}
put triangle {
opaque interior;
}
'version 6' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = 1;
center = (0,.5);
}
put triangle {
opaque exterior;
}
'version 7' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = 2;
center = (0,.5);
}
put triangle {
opaque interior;
}
'version 8' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = 2;
center = (0,.5);
}
put triangle {
opaque exterior;
}
'version 9' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .5;
center = (1,1);
}
put triangle {
opaque interior;
}
'version 10' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .5;
center = (1,1);
}
put triangle {
opaque exterior;
}
'version 11' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .25;
center = (1,1);
}
put triangle {
opaque interior;
}
'version 12' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = .25;
center = (1,1);
}
put triangle {
opaque exterior;
}
'version 13' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = 1;
center = 2;
}
put triangle {
opaque interior;
}
'version 14' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = 1;
center = 2;
}
put triangle {
opaque exterior;
}
'version 15' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = 1.5;
center = 2;
}
put triangle {
opaque interior;
}
'version 16' at (0,-4);
}
.IE
.IS
main {
put arc {
startang = 180;
endang = 0;
radius = 1.5;
center = 2;
}
put triangle {
opaque exterior;
}
'version 17' at (0,-4);
}
.IE
