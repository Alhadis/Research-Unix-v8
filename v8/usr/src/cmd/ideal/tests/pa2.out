.IS
.IE

.IS
...maxx 0
...maxy -4
...minx 0
...miny -4
...center 0 -4 'version 0
.IE
.IS
...maxx 0.25
...maxy 0.5
...minx -0.25
...miny -4
...center 0 -4 'version 1
...arc 0 0.5 -0.25 0.5 0.25 0.5 3.14159 6.28319 0.25
.IE
.IS
...maxx 0
...maxy -4
...minx 0
...miny -4
...center 0 -4 'version 2
.IE
.IS
...maxx 0.5
...maxy 0.5
...minx -0.5
...miny -4
...center 0 -4 'version 3
...arc 0 0.5 -0.5 0.5 5.96244e-09 2.77556e-17 3.14159 4.71239 0.5
...arc 0 0.5 5.96244e-09 2.77556e-17 0.5 0.5 4.71239 6.28319 0.5
.IE
.IS
...maxx 0.75
...maxy 0.5
...minx -0.75
...miny -4
...center 0 -4 'version 4
...arc 0 0.5 -0.75 0.5 -0.717707 0.282293 3.14159 3.43611 0.75
...arc 0 0.5 -0.559017 3.47522e-08 0.559017 4.80846e-08 3.87132 5.55346 0.75
...arc 0 0.5 0.717707 0.282293 0.75 0.5 5.98867 6.28319 0.75
.IE
.IS
...maxx 0.717707
...maxy 0.282293
...minx -0.717707
...miny -4
...center 0 -4 'version 5
...arc 0 0.5 -0.717707 0.282293 -0.559017 3.47522e-08 3.43611 3.87132 0.75
...arc 0 0.5 0.559017 4.80846e-08 0.717707 0.282293 5.55346 5.98867 0.75
.IE
.IS
...maxx 1
...maxy 0.5
...minx -1
...miny -4
...center 0 -4 'version 6
...arc 0 0.5 -1 0.5 -0.911438 0.0885621 3.14159 3.56562 1
...arc 0 0.5 -0.866025 1.49095e-08 0.866025 -1.70913e-07 3.66519 5.75959 1
...arc 0 0.5 0.911438 0.0885621 1 0.5 5.85915 6.28319 1
.IE
.IS
...maxx 0.911438
...maxy 0.0885621
...minx -0.911438
...miny -4
...center 0 -4 'version 7
...arc 0 0.5 -0.911438 0.0885621 -0.866025 1.49095e-08 3.56562 3.66519 1
...arc 0 0.5 0.866025 -1.70913e-07 0.911438 0.0885621 5.75959 5.85915 1
.IE
.IS
...maxx 2
...maxy 0.5
...minx -2
...miny -4
...center 0 -4 'version 8
...arc 0 0.5 -2 0.5 2 0.5 3.14159 6.28319 2
.IE
.IS
...maxx 0
...maxy -4
...minx 0
...miny -4
...center 0 -4 'version 9
.IE
.IS
...maxx 1.5
...maxy 1
...minx 0
...miny -4
...center 0 -4 'version 10
...arc 1 1 0.5 1 1.5 1 3.14159 6.28319 0.5
.IE
.IS
...maxx 0
...maxy -4
...minx 0
...miny -4
...center 0 -4 'version 11
.IE
.IS
...maxx 1.25
...maxy 1
...minx 0
...miny -4
...center 0 -4 'version 12
...arc 1 1 0.75 1 1.25 1 3.14159 6.28319 0.25
.IE
.IS
...maxx 0
...maxy -4
...minx 0
...miny -4
...center 0 -4 'version 13
.IE
.IS
...maxx 3
...maxy 1.74846e-07
...minx 0
...miny -4
...center 0 -4 'version 14
...arc 2 0 1 -8.74228e-08 3 1.74846e-07 3.14159 6.28319 1
.IE
.IS
...maxx 0
...maxy -4
...minx 0
...miny -4
...center 0 -4 'version 15
.IE
.IS
...maxx 3.5
...maxy 2.62268e-07
...minx 0
...miny -4
...center 0 -4 'version 16
...arc 2 0 0.5 -1.31134e-07 3.5 2.62268e-07 3.14159 6.28319 1.5
.IE
.IS
...maxx 0
...maxy -4
...minx 0
...miny -4
...center 0 -4 'version 17
.IE
